DWM/fix_ABOSncFiles_to_DWM.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+"""Rename ABOS files and attributes"""
+
+import warnings
+import os
+import netCDF4 as nc
+from typing import List
+from glob import glob
+
+
+# def test_replace_abos_str():
+#     string = 'my_test_XXX'
+#     old = 'XXX'
+#     new = 'YYY'
+#     expected = 'my_test_YYY'
+#     result = replace_abos_str(string, old, new)
+#     assert(result == expected)
+
+def find_att_with_str(ncobj: nc.Dataset, string: str = 'ABOS'):
+    attrs: List[str] = []
+    # for k,v in global_attr_dict:
+    #    got_requested_string = type(v) is str and string in v
+    #    if got_requested_string:
+    #        attrs.append(k)
+
+    for attname in ncobj.ncattrs():
+        attvalue = getattr(ncobj, attname)
+        got_requested_string = type(attvalue) is str and string in attvalue
+        if got_requested_string:
+            attrs.append(attname)
+    return attrs
+
+
+def replace_abos_str(string: str, old: str = 'ABOS', new: str = 'DWM'):
+    return string.replace(old, new)
+
+
+def fix_abos_attributes(ncobj: nc.Dataset, namelist: List[str], old: str = 'ABOS', new: str = 'DWM'):
+    for attname in namelist:
+        attvalue = getattr(ncobj, attname)
+        new_attvalue = replace_abos_str(attvalue, old, new)
+        setattr(ncobj, attname, new_attvalue)
+
+
+def get_netcdf_files(folder: str):
+    return glob(os.path.join(folder, '**/*.nc'), recursive=True)
```

Comment:
this is a very slow approach using the s3 mount. works, but would be better to list the files using WFS

Reply:
Yes, the problem here is that ana changed the "tmp" folder to the s3. The script is supposed to work on a temporary folder since we are doing attribute replacement.

Reply:
I didn't mention but that folder in 'dataDIR' will change, that is only a temporary address. I didn't even try to run in that path, I run it only locally with some test files